content/en/docs/setup/independent/kubelet-integration.md

```diff
@@ -96,14 +93,19 @@ such as systemd.
 
 ## Configure kubelets using kubeadm
 
-The kubeadm config API type `MasterConfiguration` embeds the kubelet's ComponentConfig under
-the `.kubeletConfiguration.baseConfig` key. Any user writing a `MasterConfiguration`
-file can use this configuration key to also set the base-level configuration for all kubelets
-in the cluster.
+It is possible to configure the kubelet that kubeadm will start if a custom `KubeletConfiguration`
+API object is passed with a configuration file like so `kubeadm ... --config some-config-file.yaml`.
+
+By calling `kubeadm config print-default --api-objects KubeletConfiguration` you can
+see all the default values for this structure.
+
+Also have a look at the [API reference for the
+kubelet ComponentConfig](https://godoc.org/k8s.io/kubernetes/pkg/kubelet/apis/kubeletconfig#KubeletConfiguration)
```

Comment:
This link is Not Found :(

Reply:
looks like it was changed to: https://godoc.org/k8s.io/kubernetes/pkg/kubelet/apis/config#KubeletConfiguration
will update.